curvefs/docker/debian10/entrypoint.sh

```diff
@@ -87,6 +87,14 @@ function prepare() {
             g_binary="$g_prefix/sbin/curve-fuse"
             g_start_args="--confPath $conf_path"
             ;;
+        monitor)
+            g_binary="python3"
+            g_start_args="target_json.py"
+            ;;
+        memcached)
+            g_prefix="/memcached"
```

Comment:
Whether memcache is part of the curvefs system (like snapshot clone server)? If so, i think place it to `/curvefs` directory is better.

Reply:
> Whether memcache is part of the curvefs system (like snapshot clone server)? If so, i think place it to `/curvefs` directory is better.

Will memcached be used by other systems in the future? Isn't this bound to curvefs?